contracts/deploy/allowListPaymaster.ts

```diff
@@ -1,10 +1,12 @@
-import { Provider, Wallet } from "zksync-web3";
 import * as ethers from "ethers";
-import { HardhatRuntimeEnvironment } from "hardhat/types";
-import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
 
+import { Provider, Wallet } from "zksync-web3";
+
+import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
Any particular reason you've updated the ordering here?

Reply:
My settings on automatically sort imports did this, reverting it back.